dotnet/src/Connectors/Connectors.Memory.Sqlite/SqliteVectorStoreRecordCollection.cs

```diff
@@ -276,29 +238,51 @@ public async IAsyncEnumerable<TRecord> GetAsync(Expression<Func<TRecord, bool>>
         SqliteFilterTranslator translator = new(this._model, filter);
         translator.Translate(appendWhere: false);
 
-        IReadOnlyList<VectorStoreRecordPropertyModel> properties = options.IncludeVectors
-            ? this._model.Properties
-            : [this._model.KeyProperty, .. this._model.DataProperties];
-
         using var connection = await this.GetConnectionAsync(cancellationToken).ConfigureAwait(false);
-        using var command = SqliteVectorStoreCollectionCommandBuilder.BuildSelectWhereCommand(
-            this._model,
-            connection,
-            top,
-            options,
-            this._dataTableName,
-            this._model.Properties,
-            translator.Clause.ToString(),
-            translator.Parameters);
+        DbCommand? command = null;
 
-        using var reader = await command.ExecuteReaderAsync(cancellationToken).ConfigureAwait(false);
-        while (await reader.ReadAsync(cancellationToken).ConfigureAwait(false))
+        if (options.IncludeVectors)
```

Comment:
this is a bug fix: we respect `IncludeVectors` now and also reuse the code with other methods that create `SELECT WHERE ` querries